rebasehelper/settings.py

```diff
@@ -28,6 +28,7 @@
 
 REBASE_HELPER_RESULTS_DIR = REBASE_HELPER_PREFIX + "results"
 REBASE_HELPER_WORKSPACE_DIR = REBASE_HELPER_PREFIX + "workspace"
+REBASE_HELPER_REBASED_SOURCES_DIR = REBASE_HELPER_PREFIX + "results/rebased_sources"
```

Comment:
Why not `REBASE_HELPER_REBASED_SOURCES_DIR = os.path.join(REBASE_HELPER_RESULTS_DIR, "rebased_sources")` just like below?